pkg/util/runtime/runtime.go

```diff
@@ -110,3 +114,27 @@ func NewServerMux() *gwruntime.ServeMux {
 	)
 	return mux
 }
+
+// InClusterBuildConfig is a helper function that builds configs by trying the InClusterConfig().
+// If InClusterConfig is unsuccessful, it falls back to BuildCustomConfigFromFlags.
+func InClusterBuildConfig(kubeconfigPath string) (*restclient.Config, error) {
+	kubeconfig, err := restclient.InClusterConfig()
+	if err == nil {
+		return kubeconfig, nil
+	}
+	klog.Warning("error creating inClusterConfig, trying BuildCustomConfigFromFlags()", err)
+	return BuildCustomConfigFromFlags("", kubeconfigPath)
+}
+
+// BuildCustomConfigFromFlags is a helper function that builds configs from a masterURL
+// or a kubeconfigPath. These parameters are passed in as command line flags for cluster
+// components. If neither masterUrl or kubeconfigPath are passed,
+// the function logs a warning and falls back to a default configuration
+func BuildCustomConfigFromFlags(masterURL, kubeconfigPath string) (*restclient.Config, error) {
```

Comment:
Small thing, but since this function isn't used anywhere else - any reason not to have the content be inline with `InClusterBuildConfig(...)` ?